Formula/pycodestyle.rb

```diff
@@ -0,0 +1,15 @@
+class Pycodestyle < Formula
+  desc "Simple Python style checker in one Python file"
+  homepage "http://pycodestyle.pycqa.org"
+  url "https://github.com/PyCQA/pycodestyle/archive/2.2.0.tar.gz"
+  sha256 "aa663451c9de97d00eff396eeffe1095fd1597491341ca3c0be54983b25b1a7d"
+  head "https://github.com/PyCQA/pycodestyle.git"
+
+  def install
+    bin.install "pycodestyle.py"
+  end
+
+  test do
+    system "#{bin}/testsuite/test_all.py"
```

Comment:
The `test` block is here to check everything’s got installed and is running correctly; not to run the project’s test suite.